plugins/quay/src/components/QuayRepository/tableHeading.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 
 import { Link, Progress, TableColumn } from '@backstage/core-components';
 
+import { Tooltip } from '@material-ui/core';
```

Comment:
If possible use the MUI 5 package, `@mui/core/Tooltip` probably.

Reply:
Quay plugin still uses `"@material-ui/core": "^4.12.2"`. I don't see any other plugins using Material UI 5 package either. Has there been an update recently to move to MUI 5? Maybe we can create a follow up ticket for this upgrade?

Reply:
Upstream will move to it eventually.  We've only just started to consider this however and are trying to stick with it for new stuff.  Totally fine to punt this concern for this PR.

Reply:
Yeah, doing the upgrade in this PR will increase the scope of this work. Will create an issue for upgrading to MUI 5.